packages/ilmomasiina-models/src/schema/eventList/index.ts

```diff
@@ -38,6 +38,11 @@ export const eventListQuery = Type.Object({
       description: "If set, only events with the provided category are included.",
     }),
   ),
+  since: Type.Optional(
+    Type.Date({
+      description: "If set, only events starting after this date are included.",
+    }),
+  ),
```

Comment:
`Type.Date` is a [typebox extension](https://github.com/sinclairzx81/typebox?tab=readme-ov-file#javascript-types), it doesn't actually parse the dates. You'll need to use `Type.String` with [`format: "date-time"`](https://json-schema.org/draft/2020-12/json-schema-validation#section-7.3.1-4.2) and parse yourself (like we do in e.g. [`updateEvent`](https://github.com/Tietokilta/ilmomasiina/blob/cf0a1fd003f235b1ed0128c38dd0de18e7f03940/packages/ilmomasiina-backend/src/routes/admin/events/updateEvent.ts#L83-L86))